README.md

```diff
@@ -80,12 +80,32 @@ resource:
       pod: true
 ```
 
+### Working with RBAC
+
+Kubernetes Engine clusters running versions 1.6 or higher introduced Role-Based Access Control (RBAC). We can create `ServiceAccount` for it to work with RBAC.
+
+```console
+$ kubectl create -f kubewatch-service-account.yaml
+```
+
+If you do not have permission to create it, you need to become a admin first. Do this:
```

Comment:
Afaics below is GKE specific, suggest clarifying this by replacing "Do this" with "For example, in GKE you would run:"